src/app/shared/entry.ts

```diff
@@ -95,7 +95,8 @@ export abstract class Entry implements OnDestroy {
   }
 
   init() {
-    this.clearState();
+    // Getting rid of this line makes the linking work again and I didn't notice any weird behaviour, but I'm not sure.. Needs more testing
```

Comment:
what's the linking work again?

Reply:
Trying to switch between different tools on the my-tools page is broken with this line. This line clears the tool after it's been selected.